template-deliverables.md

```diff
@@ -0,0 +1,145 @@
+# Template deliverables
+
+This document explains the different documentation deliverables that template authors are required to write as part of each template set. First, read our Template Contributing Guide to understand the larger context for these deliverables.
+
+## Table of contents
+
+- [Overview](#overview)
+- [Raw template](#raw-template)

```

Comment:
I think we should call this Template or Template File, for reasons I explain in more detail in a previous comment.

Reply:
Sure.

Reply:
I've adopted Adian's suggestion. Resolving.